app/views/layouts/_contact_us_link.html.haml

```diff
@@ -1,3 +1,7 @@
 - if show_contact_us_link?
-  %p
-    = link_to("Should you require help with completing this form you can contact us.", contact_us_page_path)
\ No newline at end of file
+  %details
```

Comment:
🗒  I don't if as a team we're aware of this, but this template seems to be rendered even in the main claims page, where's no form to complete, which looks weird to me 😅 .